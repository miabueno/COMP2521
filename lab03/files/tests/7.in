131
100
125
122
74
50
123
75
126
124
73
76
150
130
