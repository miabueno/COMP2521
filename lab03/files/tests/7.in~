50
73
74
75
76
100
122
123
124
125
126
130
131
150
