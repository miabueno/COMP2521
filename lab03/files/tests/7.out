Tree:
50
  \
  73
    \
    74
      \
      75
        \
        76
          \
          100
            \
            122
              \
              123
                \
                124
                  \
                  125
                    \
                    126
                      \
                      130
                        \
                        131
                          \
                          150

#nodes = 14,  #leaves = 1
Infix  : 50 73 74 75 76 100 122 123 124 125 126 130 131 150 
Prefix : 50 73 74 75 76 100 122 123 124 125 126 130 131 150 
Postfix: 150 131 130 126 125 124 123 122 100 76 75 74 73 50 
ByLevel: 50 73 74 75 76 100 122 123 124 125 126 130 131 150 
